department-of-veterans-affairs/caseflow
# Algorithm Error Handling

Connects #5833 

### Description
This PR adds error handling for the algorithm run.

### Testing Plan
1. Update the get request in line 29 of ReviewAssignmentsContainer.jsx to an invalid URL
1. Create a schedule period
1. Go to /hearings/schedule/build/upload/${id}
1. Confirm the algorithm error is displayed
1. Update the get request back to the valid URL
1. Refresh the page and confirm the page renders as expected
